docs/Manual/09 Parameters.md

```diff
@@ -72,13 +72,12 @@ Consider the parameterized nest defined above. Rather than setting each paramete
 {P0:16, P1:32, P2:16, P3:2.0}
 ```
 
-Accera provides an easy way to add all of the functions that correspond to the parameter grid at once.
+Accera provides an easy way to add all the functions that correspond to the parameter grid at once:
 ```python
 parameters = get_parameters_from_grid(parameter_grid={P0:[8,16], P1:[16,32], P2:[16], P3:[1.0,2.0]})
 package.add(nest, args=(A, B, C), base_name="matmul", parameters)
 ```
-In this case, `package.add` generates a function eight times, once for each parameter combination in the grid. Instead of `nest`, this function could take a schedule or a plan. All eight functions share the same base name, and Accera automatically adds a unique suffix to each function name to prevent duplicates.
-This pattern allows you to optionally perform filtering by inspecting the list of generated parameter values before calling `package.add`.
+In this case, `package.add` generates a function eight times, once for each parameter combination in the grid. Instead of `nest`, this function can take a schedule or a plan. All eight functions share the same base name. However, Accera automatically adds a unique suffix to each function name to prevent duplication. This pattern allows optional filtering by inspecting the generated parameter values list before calling `package.add`.
```

Comment:
In light of your offline question, do you think it's helpful to replace "Instead of `nest`, this function can take a schedule or a plan." with something like this? Feel free to edit / change for clarity.

"Instead of `nest`, `package.add` can alternatively accept a `Schedule` (if we are performing schedule transformations), or a `Plan` (if we are setting target-specific options)."

Note that a `Schedule` and a `Plan` are capitalized because we are referring them by their class names, where as `nest` is referring to the instance of `Nest` from our snippet above.

Reply:
I have used this: 
Other than `nest`, `package.add` can alternatively accept a Schedule (if we are performing schedule transformations), or a Plan (if we are setting target-specific options).